provisioning/playbook.yml

```diff
@@ -20,6 +20,7 @@
         - "{{ config_dir }}/config.yml"
         - "{{ config_dir }}/local.config.yml"
         - "{{ config_dir }}/secrets.yml"
+        - "{{ config_dir }}/{{ drupalvm_host_os|default(ansible_system.lower()) }}.config.yml"
```

Comment:
`ansible_system` doesnt entirely match Vagrant, but it still returns the main ones `windows`, `linux` and `darwin`. http://stackoverflow.com/questions/31107019/what-are-the-potential-results-for-platform-system